Latex-Briefvorlage.tex

```diff
@@ -40,6 +40,10 @@
 
 % Zeichenkodierung des Dokuments ist in UTF-8
 \usepackage[utf8]{inputenc}
+% Schriftart ist Latin Modern
+\usepackage{lmodern}
+% Zeichensatz der Ausgabe angepasst für westeuropäische Länder
+\usepackage[T1]{fontenc}
```

Comment:
Umlaute und das scharfe S werden auch so fehlerfrei ausgegeben. Was bringt diese Anweisung?

Reply:
Siehe dazu: https://texfaq.org/FAQ-why-inp-font

Reply:
Umlaute in der PDF-Datei sehen zwar richtig aus, sind aber eigentlich aus z.B. einem a und zwei Punkten zusammengesetzt. D.h. wenn man Text aus der PDF-Datei herauskopiert, kopiert man zwei Charakter (das a und zwei Punkte). Wenn man dann das Wort in einem anderen Editor einfügt, erhält man sowas wie ha``tte statt hätte. Die Anweisung T1 zu verwenden bewirkt, dass echte Umlaute ausgegeben werden. Bei deutschsprachigen LaTeX Dokumenten eigentlich immer zu empfehlen.

Reply:
Das ist bei mir nicht der Fall. Wenn ich allerdings `\usepackage[T1]{fontenc}` einfüge wird meine Schrift viel hässlicher und pixeliger, wofür man wohl wieder `\usepackage{lmodern}` bräuchte.